pyproject.toml

```diff
@@ -1,11 +1,19 @@
+[tool.poetry]
+name = "embedchain"
+version = "0.0.21"
```

Comment:
Does it mean that we will have to update this manually every time we push a new package version?

Reply:
Change Version only in pyproject.toml file and it will be read for __version__ and in setup.py file too